src/main/java/jenkins/plugins/openstack/compute/JCloudsComputer.java

```diff
@@ -39,6 +41,7 @@
     private static final Logger LOGGER = Logger.getLogger(JCloudsComputer.class.getName());
     private final ProvisioningActivity.Id provisioningId;
     private volatile AtomicInteger used = new AtomicInteger(0);
+    private transient long connectionTime;
```

Comment:
Please rename the variable to `connectedTime` so does not get confused for connection duration.